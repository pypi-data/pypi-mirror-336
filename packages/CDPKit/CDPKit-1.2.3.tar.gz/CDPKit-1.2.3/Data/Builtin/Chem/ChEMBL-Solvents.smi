WATER	[OH2]
DICHLOROMETHANE	ClCCl
TRICHLOROMETHANE	ClC(Cl)Cl
ETHYL ACETATE	CCOC(=O)C
METHANOL	CO
PROPAN-2-OL	CC(C)O
ACETONE	CC(=O)C
DMSO	CS(=O)C
ETHANOL	CCO
