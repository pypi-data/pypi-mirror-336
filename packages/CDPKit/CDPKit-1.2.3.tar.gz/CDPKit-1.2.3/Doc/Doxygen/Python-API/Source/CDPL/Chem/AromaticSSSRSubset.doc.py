#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
#

##
# \brief Implements the extraction of the aromatic rings in the SSSR of a molecular graph.
# 
class AromaticSSSRSubset(FragmentList):

    ##
    # \brief Constructs an empty <tt>AromaticSSSRSubset</tt> instance.
    # 
    def __init__() -> None: pass

    ##
    # \brief Initializes a copy of the \e %AromaticSSSRSubset instance \a arom_sssr.
    # \param arom_sssr The \e %AromaticSSSRSubset instance to copy.
    # 
    def __init__(arom_sssr: AromaticSSSRSubset) -> None: pass

    ##
    # \brief Construct a <tt>AromaticSSSRSubset</tt> instance that contains all aromatic rings of the SSSR of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to extract the aromatic rings.
    # 
    def __init__(molgraph: MolecularGraph) -> None: pass

    ##
    # \brief Replaces the current state of \a self with a copy of the state of the \c %AromaticSSSRSubset instance \a arom_sssr.
    # \param arom_sssr The \c %AromaticSSSRSubset instance to copy.
    # \return \a self
    # 
    def assign(arom_sssr: AromaticSSSRSubset) -> AromaticSSSRSubset: pass

    ##
    # \brief Replaces the current set of rings by the aromatic rings in the SSSR of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to extract the aromatic rings.
    # 
    def extract(molgraph: MolecularGraph) -> None: pass
