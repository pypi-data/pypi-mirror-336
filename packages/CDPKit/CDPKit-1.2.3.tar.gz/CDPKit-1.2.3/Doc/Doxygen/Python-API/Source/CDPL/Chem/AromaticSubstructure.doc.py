#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
#

##
# \brief Implements the perception of aromatic atoms and bonds in a molecular graph.
# 
class AromaticSubstructure(Fragment):

    ##
    # \brief Constructs an empty <tt>AromaticSubstructure</tt> instance.
    # 
    def __init__() -> None: pass

    ##
    # \brief Construct a <tt>AromaticSubstructure</tt> instance that consists of the aromatic atoms and bonds of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to perceive aromatic atoms and bonds.
    # 
    def __init__(molgraph: MolecularGraph) -> None: pass

    ##
    # \brief Replaces the currently stored atoms and bonds by the set of aromatic atoms and bonds of the molecular graph <em>molgraph</em>.
    # 
    # \param molgraph The molecular graph for which to perceive aromatic atoms and bonds.
    # 
    def perceive(molgraph: MolecularGraph) -> None: pass
