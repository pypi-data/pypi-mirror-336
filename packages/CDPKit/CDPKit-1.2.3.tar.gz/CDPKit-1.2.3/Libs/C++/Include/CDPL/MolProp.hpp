/* 
 * MolProp.hpp 
 *
 * This file is part of the Chemical Data Processing Toolkit
 *
 * Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with this library; see the file COPYING. If not, write to
 * the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 * Boston, MA 02111-1307, USA.
 */

/**
 * \file
 * \brief A convenience header including everything that is defined in namespace CDPL::MolProp.
 */

#ifndef CDPL_MOLPROP_HPP
#define CDPL_MOLPROP_HPP

#include "CDPL/MolProp/AtomProperty.hpp"
#include "CDPL/MolProp/BondProperty.hpp"
#include "CDPL/MolProp/AtomPropertyDefault.hpp"
#include "CDPL/MolProp/MassComposition.hpp"
#include "CDPL/MolProp/ElementHistogram.hpp"
#include "CDPL/MolProp/CoordinationGeometry.hpp"
#include "CDPL/MolProp/HBondDonorAtomType.hpp"
#include "CDPL/MolProp/HBondAcceptorAtomType.hpp"
#include "CDPL/MolProp/AtomFunctions.hpp"
#include "CDPL/MolProp/BondFunctions.hpp"
#include "CDPL/MolProp/AtomContainerFunctions.hpp"
#include "CDPL/MolProp/BondContainerFunctions.hpp"
#include "CDPL/MolProp/MolecularGraphFunctions.hpp"
#include "CDPL/MolProp/TPSACalculator.hpp"
#include "CDPL/MolProp/AtomHydrophobicityCalculator.hpp"
#include "CDPL/MolProp/LogSCalculator.hpp"
#include "CDPL/MolProp/XLogPCalculator.hpp"
#include "CDPL/MolProp/PEOESigmaChargeCalculator.hpp"
#include "CDPL/MolProp/MHMOPiChargeCalculator.hpp"
#include "CDPL/MolProp/HBondDonorAtomTyper.hpp"
#include "CDPL/MolProp/HBondAcceptorAtomTyper.hpp"

#endif // CDPL_MOLPROP_HPP
