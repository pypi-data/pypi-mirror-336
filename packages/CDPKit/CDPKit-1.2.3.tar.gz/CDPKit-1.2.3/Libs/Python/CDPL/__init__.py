##
# __init__.py  
#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; either
# version 2 of the License, or (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program; see the file COPYING. If not, write to
# the Free Software Foundation, Inc., 59 Temple Place - Suite 330,
# Boston, MA 02111-1307, USA.
##


from __future__ import absolute_import

import os
import sys
import platform

if sys.version_info[1] >= 8 and platform.system() == 'Windows':
    if 'site-packages' in __file__:
        os.add_dll_directory(os.path.dirname(__file__))
    else:
        os.add_dll_directory(os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), '@CDPKIT_EXECUTABLE_INSTALL_DIR@'))

from ._cdpl import *

__version__ = VERSION_STRING
