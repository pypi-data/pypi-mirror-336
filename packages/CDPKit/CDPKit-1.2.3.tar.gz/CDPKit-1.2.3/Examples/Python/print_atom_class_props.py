#!/bin/env python

##
# print_atom_class_props.py 
#
# This file is part of the Chemical Data Processing Toolkit
#
# Copyright (C) 2003 Thomas Seidel <thomas.seidel@univie.ac.at>
#
# Permission to use, copy, modify, and/or distribute this software for any purpose with
# or without fee is hereby granted.
# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD
# TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN
# NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL
# DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER
# IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
# CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
##


import sys

import CDPL.Chem as Chem
import CDPL.MolProp as MolProp


# outputs the corresponding properties of each atom of the provided molecular graph
def outputProperties(molgraph: Chem.MolecularGraph) -> None:
    Chem.calcBasicProperties(molgraph, False)               # calculate basic molecular properties (if not yet done)
    MolProp.perceiveHBondDonorAtomTypes(molgraph, False)    # perceive H-bond donor atom types and set corresponding atom properties
    MolProp.perceiveHBondAcceptorAtomTypes(molgraph, False) # perceive H-bond acceptor atom types and set corresponding atom properties

    hba_type_str = { MolProp.HBondAcceptorAtomType.UNDEF                   : 'UNDEF',
                     MolProp.HBondAcceptorAtomType.NONE                    : 'NONE',
                     MolProp.HBondAcceptorAtomType.O_H2O                   : 'O_H2O',
                     MolProp.HBondAcceptorAtomType.O_UREA                  : 'O_UREA',
                     MolProp.HBondAcceptorAtomType.O_BARBITURIC_ACID       : 'O_BARBITURIC_ACID',
                     MolProp.HBondAcceptorAtomType.O_URIC_ACID             : 'O_URIC_ACID',
                     MolProp.HBondAcceptorAtomType.O_ETHER                 : 'O_ETHER',
                     MolProp.HBondAcceptorAtomType.O_AMIDE                 : 'O_AMIDE',
                     MolProp.HBondAcceptorAtomType.O_N_OXIDE               : 'O_N_OXIDE',
                     MolProp.HBondAcceptorAtomType.O_ACID                  : 'O_ACID',
                     MolProp.HBondAcceptorAtomType.O_ESTER                 : 'O_ESTER',
                     MolProp.HBondAcceptorAtomType.O_SULFOXIDE             : 'O_SULFOXIDE',
                     MolProp.HBondAcceptorAtomType.O_NITRO                 : 'O_NITRO',
                     MolProp.HBondAcceptorAtomType.O_SELEN_OXIDE           : 'O_SELEN_OXIDE',
                     MolProp.HBondAcceptorAtomType.O_ALDEHYD               : 'O_ALDEHYD',
                     MolProp.HBondAcceptorAtomType.O_KETONE                : 'O_KETONE',
                     MolProp.HBondAcceptorAtomType.O_ALCOHOL               : 'O_ALCOHOL',
                     MolProp.HBondAcceptorAtomType.N_NH3                   : 'N_NH3',
                     MolProp.HBondAcceptorAtomType.N_DIAMINE               : 'N_DIAMINE',
                     MolProp.HBondAcceptorAtomType.N_MONO_DI_NITRO_ANILINE : 'N_MONO_DI_NITRO_ANILINE',
                     MolProp.HBondAcceptorAtomType.N_TRI_NITRO_ANILINE     : 'N_TRI_NITRO_ANILINE',
                     MolProp.HBondAcceptorAtomType.N_HALOGENO_ANILINE      : 'N_HALOGENO_ANILINE',
                     MolProp.HBondAcceptorAtomType.N_ANILINE               : 'N_ANILINE',
                     MolProp.HBondAcceptorAtomType.N_NITRILE               : 'N_NITRILE',
                     MolProp.HBondAcceptorAtomType.N_AZOLE                 : 'N_AZOLE',
                     MolProp.HBondAcceptorAtomType.N_AMINE                 : 'N_AMINE',
                     MolProp.HBondAcceptorAtomType.N_AMIDINE               : 'N_AMIDINE',
                     MolProp.HBondAcceptorAtomType.N_AZO                   : 'N_AZO',
                     MolProp.HBondAcceptorAtomType.N_AZINE                 : 'N_AZINE',
                     MolProp.HBondAcceptorAtomType.N_DIAZINE               : 'N_DIAZINE',
                     MolProp.HBondAcceptorAtomType.N_IMINE                 : 'N_IMINE',
                     MolProp.HBondAcceptorAtomType.S_SULFIDE               : 'S_SULFIDE',
                     MolProp.HBondAcceptorAtomType.S_THIOUREA              : 'S_THIOUREA',
                     MolProp.HBondAcceptorAtomType.P_MONO_DI_PHOSPHINE     : 'P_MONO_DI_PHOSPHINE',
                     MolProp.HBondAcceptorAtomType.P_TRI_PHOSPHINE         : 'P_TRI_PHOSPHINE' }

    hbd_type_str = { MolProp.HBondDonorAtomType.UNDEF                       : 'UNDEF',
                     MolProp.HBondDonorAtomType.NONE                        : 'NONE',
                     MolProp.HBondDonorAtomType.I_HI                        : 'I_HI',
                     MolProp.HBondDonorAtomType.BR_HBR                      : 'BR_HBR',
                     MolProp.HBondDonorAtomType.CL_HCL                      : 'CL_HCL',
                     MolProp.HBondDonorAtomType.S_HSCN                      : 'S_HSCN',
                     MolProp.HBondDonorAtomType.F_HF                        : 'F_HF',
                     MolProp.HBondDonorAtomType.H_H2                        : 'H_H2',
                     MolProp.HBondDonorAtomType.C_HCN                       : 'C_HCN',
                     MolProp.HBondDonorAtomType.C_ETHINE                    : 'C_ETHINE',
                     MolProp.HBondDonorAtomType.N_HN3                       : 'N_HN3',
                     MolProp.HBondDonorAtomType.N_NH3                       : 'N_NH3',
                     MolProp.HBondDonorAtomType.N_NH4                       : 'N_NH4',
                     MolProp.HBondDonorAtomType.N_AMINE                     : 'N_AMINE',
                     MolProp.HBondDonorAtomType.N_AMINIUM                   : 'N_AMINIUM',
                     MolProp.HBondDonorAtomType.N_ANILINE                   : 'N_ANILINE',
                     MolProp.HBondDonorAtomType.N_MONO_DI_NITRO_ANILINE     : 'N_MONO_DI_NITRO_ANILINE',
                     MolProp.HBondDonorAtomType.N_TRI_NITRO_ANILINE         : 'N_TRI_NITRO_ANILINE',
                     MolProp.HBondDonorAtomType.N_PYRROLE                   : 'N_PYRROLE',
                     MolProp.HBondDonorAtomType.N_AMIDE                     : 'N_AMIDE',
                     MolProp.HBondDonorAtomType.N_IMINE                     : 'N_IMINE',
                     MolProp.HBondDonorAtomType.N_IMINIUM                   : 'N_IMINIUM',
                     MolProp.HBondDonorAtomType.S_H2S                       : 'S_H2S',
                     MolProp.HBondDonorAtomType.S_HS                        : 'S_HS',
                     MolProp.HBondDonorAtomType.S_THIOL                     : 'S_THIOL',
                     MolProp.HBondDonorAtomType.O_H3PO4                     : 'O_H3PO4',
                     MolProp.HBondDonorAtomType.O_H2CO3                     : 'O_H2CO3',
                     MolProp.HBondDonorAtomType.O_HCO3                      : 'O_HCO3',
                     MolProp.HBondDonorAtomType.O_H2O2                      : 'O_H2O2',
                     MolProp.HBondDonorAtomType.O_H2O                       : 'O_H2O',
                     MolProp.HBondDonorAtomType.O_CF3SO3H                   : 'O_CF3SO3H',
                     MolProp.HBondDonorAtomType.O_HCLO4                     : 'O_HCLO4',
                     MolProp.HBondDonorAtomType.O_H2SO4                     : 'O_H2SO4',
                     MolProp.HBondDonorAtomType.O_HNO3                      : 'O_HNO3',
                     MolProp.HBondDonorAtomType.O_HSO4                      : 'O_HSO4',
                     MolProp.HBondDonorAtomType.O_HNO2                      : 'O_HNO2',
                     MolProp.HBondDonorAtomType.O_NH2OH                     : 'O_NH2OH',
                     MolProp.HBondDonorAtomType.O_H2PO4                     : 'O_H2PO4',
                     MolProp.HBondDonorAtomType.O_H3BO3                     : 'O_H3BO3',
                     MolProp.HBondDonorAtomType.O_H4SIO4                    : 'O_H4SIO4',
                     MolProp.HBondDonorAtomType.O_HPO4                      : 'O_HPO4',
                     MolProp.HBondDonorAtomType.O_H2BO3                     : 'O_H2BO3',
                     MolProp.HBondDonorAtomType.O_HO                        : 'O_HO',
                     MolProp.HBondDonorAtomType.O_SULFONIC_ACID             : 'O_SULFONIC_ACID',
                     MolProp.HBondDonorAtomType.O_MONO_DI_NITRO_PHENOL      : 'O_MONO_DI_NITRO_PHENOL',
                     MolProp.HBondDonorAtomType.O_HALOGENO_ALCOHOL          : 'O_HALOGENO_ALCOHOL',
                     MolProp.HBondDonorAtomType.O_ALCOHOL                   : 'O_ALCOHOL',
                     MolProp.HBondDonorAtomType.O_TRI_NITRO_PHENOL          : 'O_TRI_NITRO_PHENOL',
                     MolProp.HBondDonorAtomType.O_HALOGENO_PHENOL           : 'O_HALOGENO_PHENOL',
                     MolProp.HBondDonorAtomType.O_PHENOL                    : 'O_PHENOL',
                     MolProp.HBondDonorAtomType.O_CARBOXYLIC_ACID           : 'O_CARBOXYLIC_ACID',
                     MolProp.HBondDonorAtomType.O_HALOGENO_CARBOXYCLIC_ACID : 'O_HALOGENO_CARBOXYCLIC_ACID',
                     MolProp.HBondDonorAtomType.O_ENOL                      : 'O_ENOL',
                     MolProp.HBondDonorAtomType.O_OXIME                     : 'O_OXIME',
                     MolProp.HBondDonorAtomType.O_CL5_PHENOL                : 'O_CL5_PHENOL' }
    
    for atom in molgraph.atoms:
        print('- Atom #%s' % str(molgraph.getAtomIndex(atom)))
        print('\tIs std. hydrogen: %s' % str(MolProp.isOrdinaryHydrogen(atom, molgraph)))
        print('\tIs heavy atom: %s' % str(MolProp.isHeavy(atom)))
        print('\tIs unsaturated: %s' % str(MolProp.isUnsaturated(atom, molgraph)))
        print('\tIs H-bond acceptor: %s' % str(MolProp.isHBondAcceptor(atom, molgraph)))
        print('\tH-bond acceptor type: %s' % hba_type_str[MolProp.getHBondAcceptorType(atom)])
        print('\tIs H-bond donor: %s' % str(MolProp.isHBondDonor(atom, molgraph)))
        print('\tH-bond donor type: %s' % hbd_type_str[MolProp.getHBondDonorType(atom)])
        print('\tIs carbonyl carbon: %s' % str(MolProp.isCarbonylLike(atom, molgraph, True, True)))
        print('\tIs amide carbon: %s' % str(MolProp.isAmideCenter(atom, molgraph, True, True)))
        print('\tIs amide nitrogen: %s' % str(MolProp.isAmideNitrogen(atom, molgraph, True, True)))
        print('\tIs invertible nitrogen: %s' % str(MolProp.isInvertibleNitrogen(atom, molgraph)))
        print('\tIs planar nitrogen: %s' % str(MolProp.isPlanarNitrogen(atom, molgraph)))
        print('\tIs spiro center: %s' % str(MolProp.isSpiroCenter(atom, molgraph)))
        print('\tIs bridgehead: %s' % str(MolProp.isBridgehead(atom, molgraph, True)))
        
def main() -> None:
    if len(sys.argv) < 2:
        sys.exit('Usage: %s <input mol. file>' % sys.argv[0])

    # create reader for input molecules (format specified by file extension)
    reader = Chem.MoleculeReader(sys.argv[1]) 
  
    # create an instance of the default implementation of the Chem.Molecule interface
    mol = Chem.BasicMolecule()
    
    # read and process molecules one after the other until the end of input has been reached
    try:
        while reader.read(mol):
            try:
                outputProperties(mol)
            except Exception as e:
                sys.exit('Error: processing of molecule failed: ' + str(e))
                
    except Exception as e: # handle exception raised in case of severe read errors
        sys.exit('Error: reading molecule failed: ' + str(e))

    sys.exit(0)
        
if __name__ == '__main__':
    main()
