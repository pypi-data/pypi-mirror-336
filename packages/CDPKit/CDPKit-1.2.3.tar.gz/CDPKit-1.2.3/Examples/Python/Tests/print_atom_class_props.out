- Atom #0
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #1
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #2
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #3
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #4
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #5
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #6
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: True
	H-bond acceptor type: O_ALCOHOL
	Is H-bond donor: True
	H-bond donor type: O_PHENOL
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #7
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: True
	Is bridgehead: True
- Atom #8
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #9
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: True
- Atom #10
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #11
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #12
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #13
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #14
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: True
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #15
	Is std. hydrogen: False
	Is heavy atom: False
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #16
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: True
	H-bond acceptor type: O_ETHER
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #17
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: True
	H-bond acceptor type: O_ALCOHOL
	Is H-bond donor: True
	H-bond donor type: O_ALCOHOL
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #18
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #19
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: True
	H-bond acceptor type: N_AMINE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: True
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #20
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #21
	Is std. hydrogen: False
	Is heavy atom: True
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
- Atom #22
	Is std. hydrogen: True
	Is heavy atom: False
	Is unsaturated: False
	Is H-bond acceptor: False
	H-bond acceptor type: NONE
	Is H-bond donor: False
	H-bond donor type: NONE
	Is carbonyl carbon: False
	Is amide carbon: False
	Is amide nitrogen: False
	Is invertible nitrogen: False
	Is planar nitrogen: False
	Is spiro center: False
	Is bridgehead: False
